lib/src/presentation/bloc/card_bloc.dart

```diff
@@ -0,0 +1,100 @@
+import 'dart:async';
+
+import '../../core/util/card_constants.dart';
+import '../../data/model/card.dart';
+import '../../domain/repository/repository_card.dart';
+import 'interfaces/i_card_bloc.dart';
+
+class CardBloc extends ICardBloc {
+  final RepositoryCard repositoryCard;
+  final defaultEndpoint = kRacesDemonEndPointApi;
+  late int firstCardOfPage = cardBlocStartingPositionOfShowedCards;
+  late int lastCardOfPage = kCardBlocNumberOfCardsShowedPerPage;
```

Comment:
Why this two variables are defined as late?

Reply:
I already initialize them there so I don't need to declare them as late. Done.